pkg/antctl/raw/supportbundle/command.go

```diff
@@ -584,6 +590,20 @@ func controllerRemoteRunE(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to create clientset: %w", err)
 	}
 
+	if err := os.MkdirAll(option.dir, 0700|os.ModeDir); err != nil {
```

Comment:
I don't think `os.ModeDir` makes sense here
(I realize this was copied from later in the function, but we should still remove it)